src/main/scala/cromwell/engine/backend/jes/Pipeline.scala

```diff
@@ -8,11 +7,10 @@ import com.google.api.services.genomics.Genomics
 import com.google.api.services.genomics.model.CreatePipelineRequest
 import JesBackend._
 
-
-
 object Pipeline extends LazyLogging {
   def apply(command: String, workflow: WorkflowDescriptor, call: Call, jesParameters: Seq[JesParameter], projectId: String, jesConnection: JesInterface): Pipeline = {
-    logger.debug(s"Command line is $command")
+    val tag = s"JES Pipeline [UUID(${workflow.shortId}):${call.name}]"
+    logger.debug(s"$tag Command line is: $command")
```

Comment:
I like `log` personally, `Log` second.  I suspect that it probably should be `Log` according to Scala conventions?  What do you think?